config.py

```diff
@@ -9,7 +9,7 @@
 #
 
 import os
-import stat
+from stat import *
```

Comment:
The stat module does not seem to be used. Consider removing.
`import *` is considered a bad practice because it pollutes the namespace.